### ISSUE TITLE
Build ops for optimizing Policy parameters

### ISSUE THREAD

**Issue by thiagopbueno** [thiagopbueno/model-aware-policy-optimization#8]
Build ops for implementing a step of gradient descent over the Policy parameters given a TensorFlow optimizer and a batch of experiences.

**Comment by 0xangelo** [comment]
Closed after #60
